### PR TITLE
Demos/acquire open cv

### PR DESCRIPTION
**Pull request by Wovchena**

### CODE REVIEW

File: demos/acquire_opencv.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+gstreamer1.0-tools \
+libavcodec57  # libavcodec58 for UBuntu20-22 \
+libavformat57  # libavformat58 for UBuntu20-22 \
+libavresample3 \
+libavutil55 \
+libgstreamer1.0-0 \
+libswscale4
+
+### Procedure
+1. Copy OpenCV repository:
```
Comment (eaidova) [inline demos/acquire_opencv.md:61]:
```suggestion
1. Clone OpenCV repository:
```

File: demos/acquire_opencv.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+To compile application that uses OpenCV, the following environment variables should be specified:
+> export OpenCV_DIR="<OpenCV_INSTALL_DIR>/cmake"
+export LD_LIBRARY_PATH="<OpenCV_INSTALL_DIR>/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"
+export PYTHONPATH="<OpenCV_INSTALL_DIR>/python${PYTHONPATH:+:$PYTHONPATH}"
+
+
+## Building on Windows
+
+### Prerequisites 
+1. Install Microsoft Visual Studio
```
Comment (eaidova) [inline demos/acquire_opencv.md:154]: which version is supported?
e.g. openvino requires 2019

File: demos/acquire_opencv.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+## Building on Windows
+
+### Prerequisites 
+1. Install Microsoft Visual Studio 
+2. Install [cmake](https://cmake.org/download/)
+1. Install OpenVINO according to the [instruction](https://docs.openvino.ai/latest/openvino_docs_install_guides_installing_openvino_windows.html)
+2. Install [Intel® Media SDK for Windows](https://www.intel.com/content/www/us/en/developer/tools/media-sdk/choose-download-client.html)
+
+### Procedure
+1. Copy OpenCV repository:
```
Comment (eaidova) [inline demos/acquire_opencv.md:160]:
```suggestion
1. Clone OpenCV repository:
```

File: demos/acquire_opencv.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+-DOPENCV_PYTHON_INSTALL_PATH=python ^
+-DCPU_BASELINE=SSE4_2 ^
+-DOPENCV_IPP_GAUSSIAN_BLUR=ON ^
+-DWITH_INF_ENGINE=ON ^
+-DInferenceEngine_DIR="<OpenVINO_ROOT_DIRECTORY>\runtime\cmake" ^
+-Dngraph_DIR="<OpenVINO_ROOT_DIRECTORY>\runtime\cmake" ^
+-DINF_ENGINE_RELEASE=2022010000 ^
+-DVIDEOIO_PLUGIN_LIST=mfx,msmf ^
+-DCMAKE_BUILD_TYPE=Release <OpenCV_ROOT_REPO_DIRECTORY> &&
+ninja &&
```
Comment (eaidova) [inline demos/acquire_opencv.md:239]: here no ninja in prerequesits

File: demos/acquire_opencv.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+libavresample3 \
+libavutil55  # libavutil55 for Ubuntu20-22 \
+libgstreamer1.0-0 \
+libswscale4
+
+### Procedure
+1. Clone OpenCV repository:
+> git clone --recurse-submodules https://github.com/opencv/opencv.git
+2. Create build directory and enter into it:
+> mkdir ~/build-opencv && cd ~/build-opencv
```
Comment (eaidova) [inline demos/acquire_opencv.md:64]: not sure that it is good idea to build in home directory, but why not

File: demos/acquire_opencv.md (hunk 3 above)
Comment (Wovchena) [reply demos/acquire_opencv.md]: Applied

File: demos/acquire_opencv.md (hunk 4 above)
Comment (Wovchena) [reply demos/acquire_opencv.md]: Removed

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@alvoron do you have something to add or we can merge?

**Comment by alvoron** [comment]
> @alvoron do you have something to add or we can merge?

The instruction looks OK. The only potential change is the instruction location.
Yesterday Mattew or Ryan highlighted that the instruction should be in OpenCV wiki.

Isn't it a big deal to move the instruction back to OpenCV wiki if it's required by PO team?

**Comment by Wovchena** [comment]
Up to you :)
Let's just make sure that people who wanted the doc to be in OMZ are fine with the solution. Is @ilya-lavrenov ok with that?

BTW, what were Mattew or Ryan arguments to keep it in OpenCV?

**Comment by ilya-lavrenov** [comment]
> Up to you :) Let's just make sure that people who wanted the doc to be in OMZ are fine with the solution. Is @ilya-lavrenov ok with that?
> 
> BTW, what were Mattew or Ryan arguments to keep it in OpenCV?

I am Ok with having a reference from OMZ to OpenCV wiki with how to build OpenCV + OV

**Pull request closed without merging**